webpack/index.js

```diff
@@ -70,7 +70,10 @@ function getClientConfig() {
 
 function generateEnvBasedConfig() {
   if (env.isDevelopment) {
-    return {}
+    console.log(
+      chalk.red("Webpack: Compiling is not supported in development mode.")
```

Comment:
This message is technically incorrect since even in dev mode we are compiling code, through webpack dev middleware. Maybe we can expand the copy to be a bit more precise?

Reply:
```suggestion
      chalk.red("Webpack: Config build is not supported in development mode.")
```

Thanks for the correction. Since we are building a config to be [used in prod](https://github.com/artsy/force/blob/master/webpack/index.js#L140), would it be correct to log ^

Reply:
How about... 

```
Error: Attempting to create a production build while `NODE_ENV=development` is unsupported. To fix, set `NODE_ENV=production` and rerun command.
```

Reply:
As an aside, I think the work around `generateEnvBasedConfig` might still be WIP as there are a lot of production references in [this block of code](https://github.com/artsy/force/blob/2a9c220cb5490e2d3e5c28b985091eb164fdef61/webpack/index.js#L126-L141), though the comment says its only used in `dev.ts`.

Reply:
```suggestion
      chalk.red("Error: Attempting to create a production build while `NODE_ENV=development` is unsupported. To fix, set `NODE_ENV=production` and rerun the command.")
```
Thanks.